ext/lvgl/lvgl.lb

```diff
@@ -73,7 +73,7 @@ def prepare(module, options):
 
     module.depends(":platform:heap", ":cmsis:device")
 
-    return True
+    return options[":target"].identifier.platform in ["stm32", "sam"]
```

Comment:
Why? I see no reason to do that....

Reply:
Oh, I forgot LVGL also works on hosted. In that case we need to only depend on `:platform:heap` and `:cmsis:device` if the target is this, otherwise the dependencies cannot be resolved.